docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -679,8 +682,8 @@ different performance characteristics depending on the wrapped filters. This is
 fixed now, as a consequence the `or` and `and` filters are now deprecated in
 favour or `bool`.
 
-The `execution` option of the `terms` filter is now deprecated and ignored if
-provided.
+The `execution` option of the `terms` query is now removed. Additionally, `min_should_match`
+has been removed in favor of simply using a bool query.
```

Comment:
remove these notes, the only valid one at the moment is around execution, but if we remove it we do it in master and not here.